css/main.css

```diff
@@ -0,0 +1,964 @@
+    font-size: 1rem;
+    padding: 0%;
+  }
+
+  .nav-toggle {
+    display: none;
+  }
+
+  .trigger {
+    display: none;
+  }
+
+  .l-nav {
+    left: 0em;
+    width: 100%;
+    height: 3.5em;
+    background: #ffffff;
+    opacity: 1;
+    z-index: 5;
+  }
+
+  .main-nav {
+    display: block;
+  }
+
+  .nav-link {
+    display: inline-block;
+    text-align: center;
+    padding-top: 0;
+    padding: 0 1%;
+  }
+
+  .nav-link a {
+    text-decoration: none;
+    color: #2e2e2e;
+    font-size: .85em;
+    padding: 1%;
+  }
+
+  .nav-link a:active {
+    color: #fd6632;
+  }
+
+   .nav-link a:hover {
+    color: #fd6632;
+  }
+
+  .slider-break {
+    display: block;
+    width: 100%;
+  }
+
+  .slider {
+    display: none;
+  }
+
+  .hero-image {
+    background-image: url("/img/Background.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .s-hero-text {
+    padding: 0 .5em;
+    font-size: 3.25em;
+    line-height: 1.5;
+  }
+
+  .l-subnav {
+    display: block;
+  }
+
+  .l-subnav-link {
+    display: inline-block;
+  }
+
+  .s-portfolio-filter-links {
+    width: auto;
+    padding: 1%;
+  }
+
+  .s-contact-info {
+    display: inline-block;
+    background-image: url("/img/map_base.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+    margin-top: 0;
+    height: 20em;
+  }
+
+  .s-contact-info ul {
+    background-color: rgba(37, 38, 41, .8);
+    padding: .75em 1em .5em .25em;
+    margin-top: 0;
+    text-align: left;
+  }
+
+  .s-contact-info li {
+    display: inline-block;
+    padding: 0 1em 1em 0;
+    line-height: 2;
+  }
+
+  .s-contact-info li:last-child {
+    padding: 0;
+  }
+
+  .l-flex-prices {
+    width: 20%;
+    margin: 1%;
+  }
+
+  td {
+    background-color: #ffffff;
+    color: #2e2e2e;
+    font-size: 1em;
+  }
+
+  th, td {
+    padding: .25em 1em;
+    margin: 1em auto;
+  }
+
+  tr {
+    height: 2.5em;
+  }
+
+  .th-title {
+    font-size: 1.25em;
+  }
+
+  .l-svcs {
+    max-width: 30%;
+  }
+
+  .l-flex-contact {
+    text-align: right;
+    margin-left: .5em;
+    margin-right: .5em;
+  }
+
+  .l-child-m {
+    max-width: 100%;
+    padding-bottom: 0%;
+    margin-bottom: 5%;
+  }
+
+  .s-form-map-width {
+    width: 45%;
+  }
+
+  input, textarea {
+    width: 100%;
+  }
+
+  .l-team {
+    max-width: 25%;
+  }
+
+  .f-team {
+    font-size: 1.5em;
+    line-height: 1;
+    padding: 0%;
+    margin-top: 1%;
+    margin-bottom: 1%;
+  }
+
+  .l-svcs-txt-a {
+    max-width: 85%;
+  }
+
+  .l-svcs-txt-b {
+    max-width: 40%;
+  }
+
+  .l-svcs-txt-c {
+    max-width: 40%;
+  }
+
+  .l-about-txt {
+    max-width: 40%;
+    padding-bottom: 5%;
+  }
+
+  .l-client-img {
+    display: flex;
+    float: left;
+    clear: right;
+    margin: 5% auto;
+    padding: 2%;
+  }
+
+  .l-client-txt {
+    display: flex;
+    flex-flow: column;
+    text-align: left;
+    margin: 5% auto;
+    padding-left: 3%;
+  }
+
+  .l-contact-txt {
+    margin-left: 5%;
```

Comment:
Prefer the short-hand here: `margin: auto 5% 5%;` (or `margin: 0 5% 5%` if you want to ensure no top margin).